### PR TITLE
Added tests for the Label Model and the Label Controller.

### PR DESCRIPTION
**Pull request by nofaralfasi**
Added some basics tests for the Label Model and the Label Controller.
Let me know if any other tests are needed and I will add them.

### CODE REVIEW

File: test/controllers/tasks_controller_test.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 require 'test_helper'
 
 class TasksControllerTest < ActionDispatch::IntegrationTest
-  test 'should get index' do
-    get tasks_index_url
-    assert_response :success
-  end
+  # test 'should get index' do
```
Comment (stejskalleos) [inline test/controllers/tasks_controller_test.rb:4]: Nit: If the are no other tests then we should remove the file

Comment (stejskalleos) [inline test/controllers/tasks_controller_test.rb:4]: Nit: If the are no other tests then we should remove the file

**Review by stejskalleos — CHANGES_REQUESTED** [review]
One small comment otherwise ready to merge

**Review by Ron-Lavi — APPROVED** [review]
Thanks @nofaralfasi ! LGTM 👍🏼

**Pull request merged**